app/models/concerns/form526_rapid_ready_for_decision_concern.rb

```diff
@@ -111,19 +111,34 @@ def update_classification
       form526_submission_id: id
     }
 
+    Rails.logger.info(
+      "classifying 526Submission... id: #{id}, saved_claim_id: #{saved_claim_id}"
+    )
     classification = classify_by_diagnostic_code(params)
-    update_form_with_classification(classification['classification_code']) if classification.present?
+    update_form_with_classification(classification)
   end
 
   # check claims
   def classify_by_diagnostic_code(params)
     vro_client = VirtualRegionalOffice::Client.new
     response = vro_client.classify_contention_by_diagnostic_code(params)
-    response.body
+    response = response.body
+
+    if response['responseBody'] # workaround so works w/ multiple versions of the VRO code
+      response = response['responseBody']
+    end
+    response
```

Comment:
Can you elaborate a bit more on this? Out of curiosity, what are we working around?

Also, consider simplifying with a ternary or `||=`

```
response = response['responseBody']
response ||= response.body
```

or
```
response = response['responseBody'].present? ? response['responseBody'] : response.body
```

I think
```
response = response['responseBody'] || response.body
```
should also work?

Reply:
Since this looks to be the return statement, you can also get rid of the assignment to `response` if you simplify

Reply:
response.body is the hash that represents the JSON returned by the API.

Older versions of the code returned
```
{
  "statusCode": 200,
   "responseBody": {
    {
     "classification_code": <int>,
     "classification_name": <int>
   }
  }
}
```
It should now return
```
   {
     "classification_code": <int>,
     "classification_name": <int>
   }
```

Reply:
Sorry, before we resolve, would it be possible to simplify the code block? Or let me know if I missed the reason we can't

Reply:
I think something is getting missed here maybe we can hop on a short call?

We need either
`response.body["responseBody"]` or `response.body`

Depending on whether or not the contention classification endpoint is going through a java controller on the VRO side

Reply:
My bad, I realize I misread this